lib/membrane/video_compositor/video_compositor.ex

```diff
@@ -1,19 +1,18 @@
 defmodule Membrane.VideoCompositor do
   @moduledoc """
-  Membrane SDK for [VideoCompositor](https://github.com/membraneframework/video_compositor),
```

Comment:
Fixed some stuff that I noticed, but those docs could use another iteration, to both fix the language, but also to improve the explanations themselves.